modules/gapi/include/opencv2/gapi/own/convert.hpp

```diff
@@ -25,23 +25,27 @@ namespace cv
         return result;
     }
 
-           cv::gapi::own::Mat to_own(Mat&&) = delete;
+    cv::gapi::own::Mat to_own(Mat&&) = delete;
```

Comment:
...it was intentionally aligned that way, to make all `to_own` start at the same column

Reply:
...got it.

I have a general suggestion to put "inline", "static", "CV_EXPORTS*" and other modifiers and attributes on a separate line. So same column start would be preserved without playing with indentation.

The case below:

```.diff
-    inline cv::Mat to_ocv(Mat const& m) {
+    inline
+    cv::Mat to_ocv(Mat const& m) {
         return m.dims.empty()
             ? cv::Mat{m.rows, m.cols, m.type(), m.data, m.step}
             : cv::Mat{m.dims, m.type(), m.data};
     }

     cv::Mat to_ocv(Mat&&) = delete;
```